src/test/load-config-meta.spec.ts

```diff
@@ -19,8 +19,8 @@ const stencilConfig = {
   ],
 };
 
-jest.mock('glob', () => ({
-  globSync: () => ['/mock-path/stencil.config.ts'],
```

Comment:
Can we add a test where this returns `undefined`, so we can assert [this text](https://github.com/ionic-team/stencil-playwright/pull/39/files#diff-cded22c60f78b4ca99d09ebc44511c37d4ca8ac8f6fdb6bdfc57a4679ea697e0R62) gets shown with a `console.warn` spy?

Reply:
Managed to get this working. You just need to defer the call to the `jest.fn()` in the mock setup so it looks like `thingToMock: () => () => myMockedFn()`. This way the function is defined at execution time even tho the mock gets hoisted before the function declaration.